src/components/PupilViews/PupilExperience/PupilExperience.view.tsx

```diff
@@ -58,6 +59,9 @@ export const PupilPageContent = ({
     isLegacy,
   } = curriculumData;
 
+  const starterQuizNumQuestions = getInteractiveQuestions(starterQuiz).length;
+  const exitQuizNumQuestions = getInteractiveQuestions(exitQuiz).length;
```

Comment:
Should we also exclude `match` and  `order` from the count until they've been implemented?